test.py

```diff
@@ -84,6 +84,75 @@ def test_divisions_becomes_present(edap):
     assert not edap.org_unit_exists("divisions")
 
 
+def test_special_becomes_present(edap):
+    assert not edap.org_unit_exists("special")
+    edap.create_org_unit("special")
+    assert edap.org_unit_exists("special")
+
+
+def test_presidium_special_becomes_present(edap):
+    description = b'presidium'
+    presidium = "presidium"
+    special = "special"
+    # print("test_presidium_special_becomes_present begin:", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    # print("edap.object_exists_at", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup", "=", edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup"))
+    assert not edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    edap.create_special(presidium, display_name=presidium)
+    # print("edap.object_exists_at", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup", "=", edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup"))
+    assert edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    res = edap.get_special(presidium)
+    # print("edap.get_special(presidium)", res)
+    assert description in res['description']
+
+
+def test_ddea_becomes_present(edap):
+    assert not edap.org_unit_exists("ddea")
+    edap.create_org_unit("ddea")
+    assert edap.org_unit_exists("ddea")
+
+
+def test_it_ddea_becomes_present(edap):
+    description = b'it'
+    assert not edap.object_exists_at(f"cn=it,{edap.DDEA_GROUP}", "posixGroup")
+    edap.create_ddea("it", display_name="it")
+    assert edap.object_exists_at(f"cn=it,{edap.DDEA_GROUP}", "posixGroup")
+    res = edap.get_ddea("it")
+    print("edap.get_ddea(it)", res)
+    assert description in res['description']
+
+
+def test_cdea_becomes_present(edap):
+    assert not edap.org_unit_exists("cdea")
+    edap.create_org_unit("cdea")
+    assert edap.org_unit_exists("cdea")
+
+
+def test_cz_cdea_becomes_present(edap):
+    description = b'cz'
+    assert not edap.object_exists_at(f"cn=cz,{edap.CDEA_GROUP}", "posixGroup")
+    edap.create_cdea("cz", display_name="cz")
```

Comment:
It is a good idea to test with different machine name and description, so the test can notice if the underlying code swaps them by accident.

Reply:
Can you make an example?
Thanks

Reply:
Sure, for example

```
edap.create_cdea("cz", display_name="Czechia")
...
assert "Czechia" == res['description']  # i.e. "cz" != res['description']
```

If you extend the test like this, then you catch incorrect handling of input arguments.